server/routes/journeys/dailySchedule/handlers/downloadCsvHandler.ts

```diff
@@ -47,9 +47,22 @@ export default class DownloadCsvHandler implements PageHandler {
         'Appointment subtype': a.appointmentSubtypeDescription || '',
         'Room location': a.appointmentLocationDescription,
         'Court or probation team': a.externalAgencyDescription || '',
-        'Video link': a.videoLink || '',
+        'Video link': this.toFullCourtLink(a) || '',
```

Comment:
This does not need to go behind a feature toggle.  It is only ever present when the toggle is on.